modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceSettings.testcase

```diff
@@ -19,21 +19,46 @@ definition {
 	test BulkDeleteCountries {
 		property portal.acceptance = "false";
 		property test.name.skip.portal.instance = "CPCommerceSettings#BulkDeleteCountries";
+		task ("Given a Admin goes to Countries"){
+			CommerceNavigator.gotoPortlet(
+				category = "Settings",
+				portlet = "Countries");
+		}
 
-		CommerceNavigator.gotoPortlet(
-			category = "Settings",
-			portlet = "Countries");
+		task ("And verifies if the Trash icon is not present in the header"){
+			AssertElementNotPresent(locator1 = "Button#TRASH_ENABLED");
+		}
+		
+		task ("When the admin selects one country using the checkbox and verifies if the Trash icon is now present in the header"){
+			CommerceNavigator.searchEntry(entryName = "Qatar");
+
+			Click.clickNoMouseOverNoWaitForVisible(
+					key_productName = "Qatar",
+					locator1 = "CommerceEntry#PRODUCT_ENTRY_CHECKBOX");
+			AssertElementPresent(locator1 = "Button#TRASH_ENABLED");
+		}
 
-		AssertElementPresent(locator1 = "CommerceEntry#ROW_VERTICAL_ELLIPSIS");
+		task ("Then the admin can remove the country by clicking on the Trash icon"){
 
-		Pagination.changePagination(itemsPerPage = "60");
+			ClickNoError(locator1 = "Button#TRASH_ENABLED");
 
-		while (IsElementPresent(locator1 = "CommerceEntry#ROW_VERTICAL_ELLIPSIS")) {
-			CommerceProducts.deleteEntries(deleteMessage = "Are you sure you want to delete the selected countries?");
+			AssertConfirm(value1 = "Are you sure you want to delete the selected countries?");
 		}
 
-		AssertElementNotPresent(locator1 = "CommerceEntry#ROW_VERTICAL_ELLIPSIS");
-	}
+		task ("And the admin can delete all the orders with one click by using the select all checkbox"){
+			CommerceNavigator.searchEntry(entryName = "Qatar");
+
+			AssertElementNotPresent(
+					key_entryName = "Qatar",
+					locator1 = "CommerceNavigation#ENTRY_NAME");
+
+			CommerceNavigator.searchEntry(entryName = "");
```

Comment:
Can you put these lines in another task?
Here I think you are checking if the entry Qatar is still present or not.
Thanks!